doc/Caliptra.md

```diff
@@ -881,36 +881,32 @@ Caliptra contains 32 PCR 384 banks that are extendable by the SHA engine, and re
 
 |**PCR Number**|**Type**|**Extend Ctrl**|**Description**|
 | - | - | - | - |
-|PCR0|Cumulative|ROM|Holds Caliptra's FMC firmware update journey; including state and FMC, and measurements extended by ROM.|
-|PCR1|Current|ROM|Caliptra’s boot FMC, and and state. This is a snapshot of the current state.|
-|PCR2|Cumulative|FMC|Holds Caliptra's runtime firmware update journey; including state and FMC, and measurements extended by FMC.|
-|PCR3|Current|FMC|Caliptra’s boot and runtime firmware measurement and state.  This is a snapshot of the current state.|
-|PCR4|Cumulative|RT - PA1||
-|PCR5|Current|RT - PA1||
-|PCR6|Cumulative|RT - PA2||
-|PCR7|Current|RT - PA3||
-|PCR8|Cumulative|RT - PA3||
-|…|-|-||
+|PCR0|Current|ROM|Holds Caliptra’s FMC measurement and ROM policy configuration.|
+|PCR1|Cumulative|ROM|Holds journey of Caliptra’s FMC measurement and ROM policy configuration.|
+|PCR2|Current|FMC|Holds Caliptra's runtime firmware and firmware manifest measurements.|
+|PCR3|Cumulative|FMC|Holds journey of Caliptra's runtime firmware and firmware manifest measurements.|
+|PCR4 to PCR30|-|RT|Holds measurements that SoC has asked RT firmware to stash.|
+|PCR31|Cumulative|ROM|Holds journey of measurements that SoC has asked ROM to stash.|
```

Comment:
nit: I think PCR31 will hold all measurements delivered by STASH_MEASUREMENT (regardless of whether they go to ROM or RT). PCR 4-30 will hold measurements from ExtendPcr.

Reply:
will update. do we need commands to clear and lock those PCRs?

Reply:
I haven't heard anyone say that they want these PCRs to be clearable (other than by cold reset). So I think we can leave that off for now.